core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/AutoTuner.scala

```diff
@@ -647,9 +647,9 @@ class AutoTuner(
           if (getPropertyValue("spark.sql.adaptive.coalescePartitions.minPartitionNum").isEmpty) {
             // The ideal setting is for the parallelism of the cluster
             val numCoresPerExec = calcNumExecutorCores
-            val numExecutorsPerWorker = clusterProps.gpu.getCount
-            val numWorkers = clusterProps.system.getNumWorkers
-            val total = numWorkers * numExecutorsPerWorker * numCoresPerExec
+            val numExecutorsPerExecutor = clusterProps.gpu.getCount
```

Comment:
This variable `numExecutorsPerExecutor` naming a bit confusing to me, but I am fine with keeping it.

Reply:
Refactored to `numGpusPerExecutor`